### PR TITLE
Space added to make main content visible

### PR DESCRIPTION
**Pull request by ShishuSidharth**
Please merge my request for hacktoberfest

### CODE REVIEW

**Comment by ShishuSidharth** [comment]
Please merge my request if everything is ok or u can ask me to make any changes if necessary

File: src/Header.css [hunk 1]
```diff
@@ -6,6 +6,7 @@ nav {
   align-items: center;
   width: 100%;
   background-color: slategrey;
+  margin-bottom: 30px;
```
Comment (YuriDevAT) [inline src/Header.css:9]: Is there a need for margin-bottom on the header? I cannot see any changes when removing it.

**Review by YuriDevAT — CHANGES_REQUESTED** [review]

File: src/Header.css (hunk 1 above)
Comment (ShishuSidharth) [reply src/Header.css:9]: Okay I m removing that

**Review by ShishuSidharth — COMMENTED** [review]
@i have made changes

**Review by YuriDevAT — APPROVED** [review]
Seems good to me 👍🏽

**Pull request merged**